utils/tfrecords/converter.md

```diff
@@ -0,0 +1,77 @@
+# Utility for converting CVAT XML annotation file to TFRECORDS format
+
+## Description
+
+Given a CVAT XML and a directory with the image dataset, this script reads the CVAT
+XML and writes the annotations in tfrecords format into a given directory in addition
+to the label map required for the tensorflow object detection API.
+
+This implementation supports **annotated images only**. Make sure to dump the
+**XML annotations and NOT interpolations** from CVAT.
+
+
+## Installation
+
+The conversion script depends on the tensorflow object detection API, 
+for installation steps.
+
+### 1. Install necessary packages (including tensorflow).
+
+```bash
+sudo apt-get update
+sudo apt-get install -y --no-install-recommends python3-pip python3-dev
+```
+
+``` bash
+pip3 install -r requirements.txt
+```
+
+### 2. Install the tensorflow object detection API
+ If it's already installed you can check your `$PYTHONPATH`and move on to the usage section. 
+ Here's a quick (unofficial) guide on how to do that.
+ For more details follow the official guide
+ [INSTALL TENSORFLOW OBJECT DETECTION API](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/installation.md).
+ 
+```bash
+# clone the models repository
+git clone https://github.com/tensorflow/models.git
+```
+```bash
+# install some dependencies
+pip3 install --user Cython
+pip3 install --user contextlib2
+pip3 install --user pillow
+pip3 install --user lxml
+pip3 install --user jupyter
+pip3 install --user matplotlib
+```
+```bash
+# clone and compile the cocoapi
+git clone https://github.com/cocodataset/cocoapi.git
```

Comment:
How is cocoapi used?

Reply:
The cocoapi is being used in the compilation process of the object detection API which doesn't seem to compile without it. I don't know the details but It's part of the guide [INSTALL TENSORFLOW OBJECT DETECTION API](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/installation.md).